src/applications/personalization/view-dependents/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "appName": "View Dependents",
+  "entryFile": "./view-dependents-entry.jsx",
+  "entryName": "disability-my-rated-disabilities",
+  "rootUrl": "/disability/view-dependents",
```

Comment:
This root url will probably change pending a review from content/IA. For clarification, are we assuming that all dependents work falls under the umbrella of disabilities?

Reply:
Yes, this was just the URL that Jason and I picked to get something up and running but will likely change.